tests/test_galerkin.py

```diff
@@ -7,7 +7,7 @@
 
 
 @pytest.mark.parametrize("order", [1, 2, 3])
-@pytest.mark.parametrize("basis_type", ["Lagrange", "Bernstein", "integral"])
+@pytest.mark.parametrize("basis_type", ["Lagrange", "Bernstein", "integral", ("enriched", "spectral")])
```

Comment:
What's the right incantation to have this test an enriched version that uses `FIAT.GaussRadau`?

Reply:
This is the cPG quadrature variant to produce something that is exactly the RadauIIA RK method.